cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -166,18 +191,19 @@ def test_tenant_quota_enforce_via_lifecycle_infra(appliance, provider, set_roott
 # indirect is the list where we define which fixtures are to be passed values indirectly.
 @pytest.mark.parametrize('context', [ViaSSUI, ViaUI])
 @pytest.mark.parametrize(
-    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg'],
+    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg', 'set_default'],
     [
-        [('cpu', '2'), {'hardware': {'num_sockets': '8'}}, ''],
-        [('storage', '0.01'), {}, ''],
-        [('memory', '2'), {'hardware': {'memory': '4096'}}, ''],
-        [('vm', '1'), {'catalog': {'num_vms': '4'}}, '###'],
+        [('cpu', '2'), {'hardware': {'num_sockets': '8'}}, '', False],
+        [('storage', '0.01'), {}, '', False],
+        [('memory', '2'), {'hardware': {'memory': '4096'}}, '', False],
+        [('vm', '1'), {'catalog': {'num_vms': '4'}}, '###', False],
     ],
-    indirect=['set_roottenant_quota', 'custom_prov_data'],
+    indirect=['set_roottenant_quota', 'custom_prov_data', 'set_default'],
     ids=['max_cpu', 'max_storage', 'max_memory', 'max_vms']
 )
 def test_tenant_quota_enforce_via_service_infra(request, appliance, context, set_roottenant_quota,
-                                                extra_msg, custom_prov_data, catalog_item):
+                                                extra_msg, set_default, custom_prov_data,
```

Comment:
Why is `set_default` included here? Its not used by the test, and its not actually doing any setup.

Reply:
@mshriver 
- ```set_default``` fixture is used by this test case for differentiating this test case from the test cases which are using ``` entry points``` other than default.
- Here this test case needs the ```catalog_item``` fixture and ```set_default``` is called by the ```catalog_item``` fixture.
- As I said this test case uses default entry points. So the fixture ```set_default``` returns default value of ```provisioning_entry_point```.
- ```set_default``` is parametrized for such a test cases and it differentiates the tests with default entry points from tests with custom entry points.

Reply:
Obviously from my comments I'm not too much of a fan of this structure, but I'm not going to block your PR on it. As you said above, lets continue to discuss how we can better structure this test parametrization overall.

Reply:
Sure :)
Thanks @mshriver